proposals/NNNN-test-scripts-common-modules-refactoring.md

```diff
@@ -0,0 +1,94 @@
+# Test Scripts - Refactoring of Common modules
+
+* Proposal: [SDL-NNNN](NNNN-test-scripts-common-modules-refactoring.md)
+* Author: [Dmytro Boltovskyi](https://github.com/dboltovskyi)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+[ATF test scripts](https://github.com/smartdevicelink/sdl_atf_test_scripts/) use a lot of functions in commonly used modules, like:
+
+  - data accessors: file, database
+  - functions for tables: print, copy
+  - sequences:
+    - create mobile session
+    - register/activate mobile application
+    - perform policy table update
+  - etc.
+
+All these functionality are not tests artifacts but tools that tests use.
+
+## Motivation
+
+Current downsides of common modules are:
+
+  - There is no clear structure
+  - Lack of description for huge amount of functions
+  - There is no single standard for the functions
+  - Some functions are duplicated
+  - A lot of outdated and unused code
+
+## Proposed solution
+
+The purpose of this proposal is to clean up these common modules.
+
+### Leave existing modules as is for backward compatibility
```

Comment:
@dboltovskyi 
If you insist on keeping this point in the list (however I don't see why it is mentioned as proposal is about something to change) I recommend to put it at the end and don't mark it as point for changes.

Reply:
@dev-gh I would like to keep this point

Reply:
@dev-gh Regarding the comment "What is "Common modules" 
Common modules is not part of ATF - it's part of test scripts